### PR TITLE
982 apiKeys generated as site settings

### PR DESCRIPTION
**Pull request by hwindsor**
## Pull Request Overview

- Generate frontend apikeys file from SiteSettings
- Untested right now.

### CODE REVIEW

File: app/modules/site_settings/models.py [hunk 1]
```diff
@@ -313,6 +349,24 @@ def set_key_value(cls, key, value):
...
+                f"export const transloaditService = {cls.get_value('transloaditTemplateId')};",
+                f"export const googleMapsApiKey = {cls.get_value('googleMapsApiKey')};",
+                f"export const sentryDsn = {cls.get_value('sentryDsn')};",
+                f"export const flatfileKey = {cls.get_value('flatfileKey')};",
+            ]
+
+            # write to appropriate apikeys.js depending on if we're codex or mws
+            from flask import current_app
+
+            out_filename = f"_frontend.{current_app.config.get('PROJECT_NAME').lower()}/src/constants/apiKeys.js"
```
Comment (mmulich) [inline app/modules/site_settings/models.py:366]: We want a view to produce this information. The frontend source is not available to be written to in a production deployment.

Comment (mmulich) [reply app/modules/site_settings/models.py:366]: In other words, frontend devs will change where they get these keys based on where we put the view or where it is already available.

**Review by mmulich — APPROVED** [review]
👍🏼

**Comment by brmscheiner** [comment]
I can't tell whether this PR makes the new site settings patchable, returned on `GET /block`, etc. But these do look like the right settings

**Comment by mmulich** [comment]
> I can't tell whether this PR makes the new site settings patchable, returned on `GET /block`, etc. But these do look like the right settings

I'd guess that if existing site settings are patchable, then so are these. The route added is more for general public consumption (read-only), while the existing site settings api would be for management of these values.

**Pull request merged**